doc/network-zones.md

```diff
@@ -4,57 +4,149 @@ discourse: 12033,13128
 
 # How to configure network zones
 
-Network zones are used to hold DNS records for LXD networks.
+```{note}
+Network zones are available for the {ref}`OVN network <network-ovn>` and the {ref}`Bridge network <network-bridge>`.
+```
 
-Each network can be related to up to 3 zones for:
+Network zones can be used to hold DNS records for LXD networks.
 
- - Forward DNS records
- - IPv4 reverse DNS records
- - IPv6 reverse DNS records
+If you are operating a LXD cluster with multiple instances across a large set of networks, you can use network zones to automatically maintain valid forward and reverse records for all your instances.
+Such DNS records are important not only for ease of access to the instances, but also to avoid hosted services getting flagged as potential spam due to lacking reverse DNS records.
```

Comment:
How about rephrasing slightly to avoid implying that using DNS records is required to easily access instances (which isn't true as you can use `lxc exec` without DNS records). DNS records make it easier to access network services running on the instance.

> Using DNS records are useful for ease of access to network services running on the instance.
It is also important when hosting an outbound SMTP service as without correct forward and reverse DNS entries for the instance it can cause sent mail to be flagged as potential spam.